designs/package-spec.md

```diff
@@ -0,0 +1,78 @@
+# Package Controller Configuration
+
+## Introduction
+
+**Problem:** currently the ability to configure the package controller is very limited and requires code changes
+
+Code changes currently have to be made to configure the package controller during cluster creation or using the install command. Changes need to be coordinated between the CLI and the package controller helm chart to alter configuration. This makes it more difficult to support things like various private registry configurations.
```

Comment:
Could you explain a bit more what are the "code changes" and the coordination between CLI and helm chart needed?

Reply:
Currently, when the CLI installs the package controller, we have to change code to pass those values to the helm install command(e.g.: `--set defaultRegistry=public.ecr.aws/213456789`). If we change the names of the values or add ones we have to change the CLI.

Reply:
This example is for testing, but basically most fields in our values.yaml, we'd want to have here. I've added them.